Server/User/routehelper.py

```diff
@@ -38,9 +66,9 @@ def createUser(data):
 		return response
 
 	# Verify that the user was added
-	usr2 = Users.objects.get(username=username)
+	usr2 = Users.objects.filter(username=username).first()
```

Comment:
So "get" throws an error if the user name doesn't exist, what does "filter" do instead? Worth commenting that here?